.changes/gtk-status.md

```diff
@@ -0,0 +1,5 @@
+---
+"tao": patch
+---
+
+Implement basic menu builder for macOS, Windows and Linux.
```

Comment:
Ooops! ;)

Reply:
ooops I wrote the description to status-bar.md haha